core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -197,14 +207,7 @@ class ZkUtils(val zkClient: ZkClient,
                               BrokerSequenceIdPath,
                               IsrChangeNotificationPath)
 
-  val securePersistentZkPaths = Seq(BrokerIdsPath,
```

Comment:
Sorry, you are right. I missed the fact that recursion is already in place. (1) Could you use named constant instead of string to avoid typo?  (2) Could you add a comment that future new persistent ZK paths may need to be added to this list? (3) We could probably just rename securePersistentZkPaths to securePersistentZkPathsRoot without introducing zkPathsRoot?